feature/experimental/p4rt/ate_tests/performance_test/performance_test.go

```diff
@@ -301,7 +301,7 @@ func testPktInPktOut(t *testing.T, args *testArgs) {
 		wg.Wait() // Wait for all four goroutines to finish before exiting.
 
 		// Wait for the packetOut requests to be completed on the server side
-		time.Sleep(30 * time.Second)
+		time.Sleep(60 * time.Second)
```

Comment:
time.Minute?

Reply:
Is there a condition you can wait for instead, like whether the counter is present?